From c9d0077f41945c41c62f5e62745f55e4045274d3 Mon Sep 17 00:00:00 2001
From: Chris Dimich <chris.dimich@boundarydevices.com>
Date: Mon, 14 Mar 2022 09:27:08 -0700
Subject: [PATCH] start_isp.sh: remove 4k and -lm option

4k resolution doesn't provide the best perf for the cam, 1080p60 is
definitely better for demos.
Also remove uneeded -lm option from run.sh script as not needed.

Signed-off-by: Chris Dimich <chris.dimich@boundarydevices.com>
---
 imx/start_isp.sh | 19 ++++---------------
 1 file changed, 4 insertions(+), 15 deletions(-)

diff --git a/imx/start_isp.sh b/imx/start_isp.sh
index 95cbc1993..eb0c763b8 100755
--- a/imx/start_isp.sh
+++ b/imx/start_isp.sh
@@ -18,20 +18,9 @@ if [ $NR_DEVICE_TREE_BASLER -eq 1 ]; then
 	echo "Starting isp_media_server for Single Basler daA3840-30mc"
 
 	cd $RUNTIME_DIR
-
-	if [ $NR_DEVICE_TREE_OV5640 -eq 0 ]; then
-
-		# Default configuration for Basler daA3840-30mc: basler_4k
-		# Available configurations: basler_4k, basler_1080p60, basler_4khdr, basler_1080p60hdr
-		exec ./run.sh -c basler_4k -lm
-
-	elif [ $NR_DEVICE_TREE_OV5640 -eq 1 ]; then
-
-		# Default configuration for Basler daA3840-30mc: basler_1080p60
-		# Available configurations: basler_1080p60, basler_1080p60hdr
-		exec ./run.sh -c basler_1080p60 -lm
-
-	fi
+	# Default configuration for Basler daA3840-30mc: basler_1080p60
+	# Available configurations: basler_1080p60, basler_1080p60hdr
+	exec ./run.sh -c basler_1080p60
 
 elif [ $NR_DEVICE_TREE_BASLER -eq 2 ]; then
 
@@ -40,7 +29,7 @@ elif [ $NR_DEVICE_TREE_BASLER -eq 2 ]; then
 	cd $RUNTIME_DIR
 	# Default configuration for Basler daA3840-30mc: dual_basler_1080p60
 	# Available configurations: dual_basler_1080p60, dual_basler_1080p60hdr
-	exec ./run.sh -c dual_basler_1080p60 -lm
+	exec ./run.sh -c dual_basler_1080p60
 # check if the os08a20 device has been enabled in the device tree
 elif [ $NR_DEVICE_TREE_OS08A20 -eq 1 ]; then
 
-- 
2.35.1

